cmd/thor/node/tx_stash_test.go

```diff
@@ -20,9 +19,8 @@ import (
 )
 
 func newTx() *tx.Transaction {
-	tx := new(tx.Builder).Nonce(rand.Uint64()).Build() // nolint:gosec
-	sig, _ := crypto.Sign(tx.SigningHash().Bytes(), genesis.DevAccounts()[0].PrivateKey)
-	return tx.WithSignature(sig)
+	tx, _ := new(tx.Builder).Nonce(rand.Uint64()).BuildAndSign(genesis.DevAccounts()[0].PrivateKey) // nolint:gosec
```

Comment:
Please, replace `nolint:gosec` with `#nosec`, this way is both picked up by the linter and gosec